lib/Switch.js

```diff
@@ -143,11 +142,11 @@ export class Switch extends Component {
 							{ transform: [{ translateX: transformSwitch }], width: circleSize * 2 },
 						]}
 					>
-						<Text style={[styles.text, styles.inactiveText, styles.paddingRight, activeTextStyle]}>
```

Comment:
Can you please explain this change ?

Reply:
Just removing `styles.inactiveText` which isn't defined in the StyleSheet object.